frontend/src/reviews/ReviewsListContainer.tsx

```diff
@@ -1,26 +1,23 @@
 import { Container, Divider, List, Stack } from '@mui/material'
-import router from 'next/router'
+import { useRouter } from 'next/router'
 import { useEffect, useState } from 'react'
 import SimpleListItemButton from 'src/common/SimpleListItemButton'
 import ReviewsList from 'src/reviews/ReviewsList'
-import { ReviewKind, ReviewKindKeys, ReviewListStatusKeys } from 'types/types'
-
-function isReviewListContainerCategory(value: string | string[] | undefined): value is ReviewKindKeys {
-  return value === ReviewKind.RELEASE || value === ReviewKind.ACCESS
-}
+import { isReviewKind, ReviewKind, ReviewKindKeys, ReviewListStatusKeys } from 'types/types'
 
 export interface ReviewsListContainerProps {
   status: ReviewListStatusKeys
 }
 
 export default function ReviewsListContainer({ status }: ReviewsListContainerProps) {
+  const router = useRouter()
   const { category } = router.query
 
-  const [selectedCategory, setSelectedCategory] = useState<ReviewKindKeys>(ReviewKind.RELEASE)
+  const [selectedCategory, setSelectedCategory] = useState<ReviewKindKeys>(ReviewKind.ACCESS)
 
   useEffect(() => {
-    if (isReviewListContainerCategory(category)) {
-      setSelectedCategory(category ?? ReviewKind.RELEASE)
+    if (isReviewKind(category)) {
+      setSelectedCategory(category ?? ReviewKind.ACCESS)
```

Comment:
This can be changed to `setSelectedCategory(category)` as it already has passed the `isReviewKind` check